pkg/wallet/crypto_box.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package wallet
+
+import (
```

Comment:
Needs a TODO to move out of the wallet package.

Reply:
TODO added